macaroon_pouch.go

```diff
@@ -84,6 +84,19 @@ func newMacaroonPouch(macaroonDir, customMacPath string) (macaroonPouch,
 			return nil, err
 		}
 
+		return macaroonPouch{
+			invoiceMacFilename:   mac,
+			chainMacFilename:     mac,
+			signerMacFilename:    mac,
+			walletKitMacFilename: mac,
+			routerMacFilename:    mac,
+			adminMacFilename:     mac,
+			readonlyMacFilename:  mac,
+		}, nil
```

Comment:
instead of duplicating this code, we could just declare a `mac` var above this if/else and then set its value and return if it's not empty